cumulus/polkadot-parachain/src/chain_spec/bridge_hubs.rs

```diff
@@ -51,14 +40,7 @@ impl FromStr for BridgeHubRuntimeType {
 
 	fn from_str(value: &str) -> Result<Self, Self::Err> {
 		match value {
-			polkadot::BRIDGE_HUB_POLKADOT => Ok(BridgeHubRuntimeType::Polkadot),
-			polkadot::BRIDGE_HUB_POLKADOT_LOCAL => Ok(BridgeHubRuntimeType::PolkadotLocal),
-			polkadot::BRIDGE_HUB_POLKADOT_DEVELOPMENT =>
-				Ok(BridgeHubRuntimeType::PolkadotDevelopment),
-			kusama::BRIDGE_HUB_KUSAMA => Ok(BridgeHubRuntimeType::Kusama),
-			kusama::BRIDGE_HUB_KUSAMA_LOCAL => Ok(BridgeHubRuntimeType::KusamaLocal),
-			kusama::BRIDGE_HUB_KUSAMA_DEVELOPMENT => Ok(BridgeHubRuntimeType::KusamaDevelopment),
-			westend::BRIDGE_HUB_WESTEND => Ok(BridgeHubRuntimeType::Westend),
+			westend::BRIDGE_HUB_WESTEND => unimplemented!("TODO: fix BridgeHubWestend"),
```

Comment:
@seadanda could you, please, update all those TODOs with this issue:
```suggestion
			westend::BRIDGE_HUB_WESTEND => unimplemented!("TODO: fix BridgeHubWestend: https://github.com/paritytech/parity-bridges-common/issues/2602"),
```